skyve-ejb/src/main/java/modules/admin/DataMaintenance/RefreshDocumentTuplesJob.java

```diff
@@ -89,6 +90,10 @@ public void execute() throws Exception {
 												sb.toString(),
 												dm.getRefreshOption().toLocalisedDescription(),
 												bean.getBizId()));
+						if (flagFailedData) {
```

Comment:
Hi @brandon-klar, you should not perform a boolean operation on a Boolean object. This should be `if(Boolean.TRUE.equals(flagFailedData)) {`. Can you please update it?